pom-gwt.xml

```diff
@@ -122,41 +122,16 @@
     <dependency>
       <groupId>com.google.gwt</groupId>
       <artifactId>gwt-user</artifactId>
-      <version>2.8.0-SNAPSHOT</version>
+      <version>2.8.0-beta1</version>
     </dependency>
 
     <dependency>
       <groupId>com.google.gwt</groupId>
       <artifactId>gwt-dev</artifactId>
-      <version>2.8.0-SNAPSHOT</version>
+      <version>2.8.0-beta1</version>
     </dependency>
   </dependencies>
 
-  <repositories>
```

Comment:
Yeah it's a bit confusing here... The one I deleted points to https://oss.sonatype.org/content/repositories/google-snapshots, which does host Guava.

gwt-2.8.0-beta1 uses oss-parent as the parent POM (see https://repo1.maven.org/maven2/com/google/gwt/gwt/2.8.0-beta1/gwt-2.8.0-beta1.pom):
`
<parent>
<groupId>org.sonatype.oss</groupId>
<artifactId>oss-parent</artifactId>
<version>4</version>
</parent>
`
That POM (https://repo1.maven.org/maven2/org/sonatype/oss/oss-parent/4/oss-parent-4.pom)
adds the https://oss.sonatype.org/content/repositories/snapshots repository, which actually contains Guava 20.0-SNAPSHOT.

So I guess it's fine to delete it here.